docs/wiki/basic-tutorials/interactions/responding.md

```diff
@@ -23,6 +23,9 @@ event.getInteraction()
 ::: tip INFO
 Note that you have to respond withing 3 seconds, or the command will fail. If you need longer than 3 seconds you have to
 respond with `respondLater()` which allows you to respond within 15 minutes.
+
+Because of this time limitation sending any files when using `createImmediateResponder` is not possible.
+You can use the `String` link methodes to avoid this. Or better just use `respondLater`
```

Comment:
What do you mean by "`String` link methods"?

Reply:
Fair, I meant a online resource like the discord CDN or something. Do you want me to update that?

Reply:
Please do. The current version is quite unclear but I can see the value in your addition as it's a question that will definitly come up.

Reply:
I re-pushed the branch with a new version. Can you check it. Apparently git doesn't update. For me at least.